mmv1/templates/terraform/examples/firebasehosting_channel_full.tf.erb

```diff
@@ -0,0 +1,19 @@
+resource "google_firebase_hosting_site" "default" {
+  provider = google-beta
+  project  = "<%= ctx[:test_env_vars]['project_id'] %>"
+  site_id = "<%= ctx[:vars]['site_id'] %>"
+}
+
+resource "google_firebase_hosting_channel" "full" {
+  provider = google-beta
+  project  = "<%= ctx[:test_env_vars]['project_id'] %>"
+  site_id = "<%= ctx[:vars]['site_id'] %>"
+  channel_id = "<%= ctx[:vars]['channel_id'] %>"
```

Comment:
These can be changed to references to google_firebase_hosting_site.default.site_id or similar to make us not need the explicit dependency below

Reply:
Good idea. Done.